projects/front-end-library/src/lib/_doc/styles/1-colors/colors.js

```diff
@@ -14,10 +15,20 @@ const colors = [
                 label            : 'Neutral Tertiary',
                 name             : 'neutral-tertiary'
             },
+            {
+                label            : 'Highlight',
+                name             : 'highlight',
+                hasUtilityClass  : false,
+            },
+            {
+                label            : 'Brand',
+                name             : 'brand'
+            },
         ]
     },
     {
-        name: 'Background',
+        name: 'Background ',
+        description: 'Theses background colors are meant to be used on non-interactive elements. The Highlight isn\'t compatible with the Reversed option',
```

Comment:
```suggestion
        description: 'These background colors are meant to be used on non-interactive elements.',
```
ça me parrait faux. C'Est compatible, c'est juste le même effet. Dire que quelque chose est pas compatible, c'est que ca crée un problème

Reply:
reverse + highlight produit un bug visuel où on écrit en blanc sur fond jaune. c'est incompatible point de vue UX car ça n'offre pas le contraste nécessaire. si pour toi ce n'est pas un problème, mes yeux ne partagent pas ton avis. par ailleurs il existe plusieurs bugs/tâches jira qui soulèvent cet enjeux.

Reply:
supprimé - remplacé par une description simple